src/administrative-sdk/administrative-sdk.js

```diff
@@ -23,104 +23,337 @@ export default class AdministrativeSDK {
+   */
   getChoiceRecognition(challengeId, recognitionId) {
     return this._choiceRecognitionController.getChoiceRecognition(challengeId, recognitionId);
   }
 
+  /**
+   * List all choice recognitions in a specific {@link ChoiceChallenge} from the current active {@link Organisation}
+   * derived from the OAuth2 scope.
+   *
+   * @param {ChoiceChallenge#id} challengeId - Specify a choice challenge to list speech recognitions for.
+   * @returns {Promise} Promise containing an array of ChoiceRecognitions.
+   * @throws {Promise} {@link ChoiceChallenge#id} is required.
+   * @throws {Promise} If no result could not be found.
+   */
   listChoiceRecognitions(challengeId) {
     return this._choiceRecognitionController.listChoiceRecognitions(challengeId);
   }
 
+  /**
+   * Create an organisation. The organisation will be owned by the current active tenant.
+   *
+   * @param {Organisation} organisation - Object to create.
+   * @returns {Promise} Promise containing the newly created object.
+   * @throws {Promise} If the server returned an error.
+   */
   createOrganisation(organisation) {
     return this._organisationController.createOrganisation(organisation);
   }
 
+  /**
+   * Get an organisation. You can only get an organisation the current tenant is an owner of.
+   *
+   * @param {Organisation#id} organisationId - Specify an organisation identifier.
+   * @returns {Promise} Promise containing an Organisation.
+   * @throws {Promise} {@link Organisation#id} field is required.
+   * @throws {Promise} If no result could not be found.
+   */
   getOrganisation(organisationId) {
     return this._organisationController.getOrganisation(organisationId);
   }
 
+  /**
+   * List all organisations the current tenant is owner of.
+   *
+   * @returns {Promise} Promise containing a list of Organisations.
+   * @throws {Promise} If no result could not be found.
+   */
   listOrganisations() {
     return this._organisationController.listOrganisations();
   }
 
+  /**
+   * Start a pronunciation analysis from streaming audio.
+   *
+   * @param {PronunciationChallenge} challenge - The pronunciation challenge to perform.
+   * @param {AudioRecorder} recorder - The audio recorder to extract audio from.
+   * @param {boolean} [trim] - Whether to trim the start and end of recorded audio (default: true).
+   * @returns {Promise} A {@link https://github.com/cujojs/when} Promise containing a {@link PronunciationAnalysis}.
+   * @emits {string} 'ReadyToReceive' when the call is made to receive audio. The recorder can now send audio.
+   * @emits {Object} When the sent audio has finished alignment. Aligning audio is the process of mapping the audio
+   * to spoken words and determining when what is said. An object is sent containing a property 'progress',
+   * which is the sent audio alignment, and a property 'referenceAlignment' which is the alignment of the
+   * reference audio.
+   * @throws {Promise} If challenge is not an object or not defined.
```

Comment:
I wonder how we're ever going to keep these up-to-date. But that's another issue for another day.